package.json

```diff
@@ -6,6 +6,7 @@
         "watch": "npm run development -- --watch",
         "watch-poll": "npm run watch -- --watch-poll",
         "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "lint": "tsc --noEmit --incremental false && eslint */**/*.{ts,tsx} --quiet --fix",
```

Comment:
You can run `npm run lint` to check that the typescript compiles and check if there are any lint errors. The `--fix` option means will also fix any lint errors that it's able to automatically fix. If there are some lint errors that it can't automatically fix, then it will output the errors so you can manually fix them.

Reply:
That sounds great! Thanks for adding it.